### PR TITLE
LPD-31206 - Refactor poshi test GDPR#CanDeleteASingleStagedAndLiveBlogsEntry

### PR DESCRIPTION
**Pull request by brbalazs**
https://liferay.atlassian.net/browse/LPD-31206

### CODE REVIEW

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by brbalazs** [comment]
ci:test:playwright:nocompile

**Comment by brbalazs** [comment]
ci:test:playwright:nocompile

**Comment by brbalazs** [comment]
ci:test:playwright:nocompile

**Comment by brbalazs** [comment]
ci:test:playwright:nocompile

**Review by smottal — APPROVED** [review]

File: modules/test/playwright/tests/users-admin-web/gdpr.spec.ts [hunk 1]
```diff
@@ -299,3 +306,130 @@ testAdmin.describe('LPD-27068 Refactor of GDPR#CanAnonymizeAllEntries', () => {
...
+		});
+
+		await apiHelpers.headlessAdminUser.postRoleByExternalReferenceCodeUserAccountAssociation(
+			'Administrator',
+			userAccount.id
+		);
+
+		await performLogout(page);
+		await performLogin(page, userAccount.alternateName);
+
```
Comment (smottal) [inline modules/test/playwright/tests/users-admin-web/gdpr.spec.ts:352]: Don't we need to put a try/finally to do the "login as admin" in the finally?

File: modules/test/playwright/tests/users-admin-web/gdpr.spec.ts [hunk 2]
```diff
@@ -299,3 +306,130 @@ testAdmin.describe('LPD-27068 Refactor of GDPR#CanAnonymizeAllEntries', () => {
...
+			)
+		).click();
+
+		await usersAndOrganizationsPage.deletePersonalDataMenuItem.click();
+
+		await expect(
+			personalDataErasurePage.selectAllItemsOnPageCheckbox
+		).toBeVisible();
+
+		await page.getByRole('link', {name: '6'}).click();
```
Comment (smottal) [inline modules/test/playwright/tests/users-admin-web/gdpr.spec.ts:403]: ??? ... this is a locator and should go in the page AND ... we have a link called 6? is the pagination?

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by brbalazs** [comment]
ci:test:sf

File: modules/test/playwright/tests/users-admin-web/gdpr.spec.ts [hunk 3]
```diff
@@ -299,3 +306,130 @@ testAdmin.describe('LPD-27068 Refactor of GDPR#CanAnonymizeAllEntries', () => {
 		}
 	);
 });
+
+testAdmin(
+	'LPD-31206 - Can delete a single staged and live blogs entry',
```
Comment (smottal) [inline modules/test/playwright/tests/users-admin-web/gdpr.spec.ts:311]: In a next PR can you update this to remove the - (I think we are not using it)

**Review by smottal — APPROVED** [review]

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:forward

File: modules/test/playwright/tests/users-admin-web/gdpr.spec.ts (hunk 3 above)
Comment (brbalazs) [reply modules/test/playwright/tests/users-admin-web/gdpr.spec.ts:311]: I'll change it next time, thanks!

**Pull request closed without merging**